src/@apollo/client/core/ApolloClient__Core_WatchQueryOptions.res

```diff
@@ -395,7 +424,7 @@ module MutationOptions = {
     errorPolicy: option<ErrorPolicy.t>,
     optimisticResponse: option<'jsVariables => 'data>,
```

Comment:
shouldn't this be `option<'variables => 'data>`?

Reply:
Yes, that would be ideal, but the last I knew, `graphql-ppx` did not expose `parseVariables` functionality: https://github.com/reasonml-community/graphql-ppx/issues/220

Has that changed?

Reply:
Ah yeah it looks like `graphql-ppx` does not yet do this looking at the .bs.js in EXAMPLES. I needed to use that for #128 so that's a shame

Reply:
@jfrolich, you mentioned a `parseVariables` in #128. Are we missing it somewhere?

Reply:
@jeddeloh you ok with dropping #128 from this PR for now?

Reply:
@illusionalsagacity It feels like there still a lot of good additions here? What do you think about just dropping the `parseVariables` from `MutationUpdaterFunction? It's not ideal and causes questions like you brought up here: https://github.com/jeddeloh/rescript-apollo-client/pull/138/files#diff-f540d981363a00a06bb56b829590b3125ec97d1279f7eeff06622150afffbfb3R908 but it's at least serviceable.

Reply:
@jeddeloh yeah, I was just thinking of dropping [this commit](https://github.com/jeddeloh/rescript-apollo-client/pull/138/commits/ed6839ca2bc2807704dd8df52b42d26811c5114d) and opening a new draft PR for that one

Reply:
Sounds good!